layers/+lang/typescript/packages.el

```diff
@@ -72,6 +72,8 @@
                 (tide-setup)
                 (flycheck-mode +1)
                 (eldoc-mode +1)
+                (define-key spacemacs-web-mode-map (kbd "g g") 'tide-jump-to-definition)
+                (define-key spacemacs-web-mode-map (kbd "g u") 'tide-references)
```

Comment:
@TheBB  I noticed that `tide-jump-to-definition` is already in `spacemacs-jump-handlers-typescript-mode` but it can't jump around. I could call the `tide-jump-to-definition` manually to jump around. Even if I add `:async t` to the jump handlers, it still doesn't work.

Any suggestions? thanks.